packages/flutter_tools/test/general.shard/emulator_test.dart

```diff
@@ -20,106 +20,252 @@ import '../src/common.dart';
 import '../src/context.dart';
 import '../src/mocks.dart';
 
+const FakeEmulator emulator1 = FakeEmulator('Nexus_5', 'Nexus 5', 'Google');
```

Comment:
I hoisted the logic out of the MockProcessManager in favor of listing out the exact commands with the FakeProcessManager